### PR TITLE
use TSS2 long tuning on all Toyotas

### PR DESCRIPTION
**Pull request by pd0wm**

### CODE REVIEW

**Comment by ntegan1** [comment]
all toyotas without a pedal :(

File: selfdrive/car/toyota/interface.py [hunk 1]
```diff
@@ -320,8 +320,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.longitudinalTuning.kpV = [1.2, 0.8, 0.765, 2.255, 1.5]
```
Comment (cydia2020) [inline selfdrive/car/toyota/interface.py:320]: Need to change here as well.

**Comment by cydia2020** [comment]
Just tested this, doesn't work well on my Prius 2018, constantly overshoots and undershoots

**Comment by pd0wm** [comment]
Do you have a drive I can look at? With or without pedal? Still need to update the tuning for the pedal.

**Comment by cydia2020** [comment]
> Do you have a drive I can look at? With or without pedal? Still need to update the tuning for the pedal.

Pedal, but min acc speed is 5m/s for me on my fork so it shouldn't matter, i only left the TSS2.0 long tuning for the test, so I wasn't using the pedal tuning for the test, so for the oscillation issue I think we can ignore the pedal for now.

I'll put a drive here hopefully by tomorrow.

**Pull request closed without merging**